src/main/java/io/jenkins/plugins/fontawesome/FontAwesomeIcons.java

```diff
@@ -0,0 +1,110 @@
+package io.jenkins.plugins.fontawesome;
+
+import org.apache.commons.lang3.StringUtils;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.file.*;
+import java.util.*;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.stream.Stream;
+
+/**
+ * Utility to work with icons provided by the font-awesome-api-plugin.
+ *
+ * @author strangelookingnerd
+ */
+public final class FontAwesomeIcons {
+
+    private static final Logger LOGGER = Logger.getLogger(FontAwesomeIcons.class.getName());
+    private static final String SVG_FILE_ENDING = ".svg";
+    private static final String IMAGES_SYMBOLS_PATH = "images/symbols/";
+    private static final String FONT_AWESOME_API_PLUGIN = "font-awesome-api";
+    private static final String ICON_CLASS_NAME_PATTERN = "symbol-%s plugin-" + FONT_AWESOME_API_PLUGIN;
+
```

Comment:
Can you please move all public code here?

Reply:
Not sure what you mean by this. Should I make these constants public?